automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -361,9 +363,16 @@ def sync_capsule_repos_to_upgrade(capsules):
 
     CAPSULE_URL
         The url for capsule repo from latest satellite compose.
+        Optional, the TO_VERSION Caspule repo from Redhat Repos will be used
```

Comment:
It should be rephrased like: If not provided, capsule repo from Red Hat repositories would be enabled.